src/main/java/seedu/address/logic/commands/FindByTagTomorrowCommand.java

```diff
@@ -18,7 +18,7 @@
     public static final String COMMAND_WORD = "c-tomorrow";
 
     public static final String MESSAGE_SUCCESS = "Tomorrow is %2$s."
-            + "\nThere are total %1$s employees working tomorrow."
+            + "\nThere are total %1$s employees available tomorrow."
```

Comment:
Should be phrased as "There are a total of ... "

Reply:
Thanks for the suggestion. will edit it in the latest commit.